sys/include/suit/v4/suit.h

```diff
@@ -91,74 +104,191 @@ typedef enum {
+ *
+ * @return              0 when the iterator is already at the end of the container
+ * @return              the number of returned (key, value) pair, e.g. 1
+ */
+int suit_cbor_map_iterate(CborValue *it, CborValue *key, CborValue *value);
 
-int suit_cbor_get_int(const CborValue *key, int *out);
-int suit_cbor_get_uint(const CborValue *key, unsigned *out);
+/**
+ * @brief Get cbor value as int
+ *
+ *
+ * @param[in]   it      cbor container iterator
+ * @param[out]  out     address of the returned integer
+ *
+ * @return              SUIT_OK on success
+ * @return              SUIT_ERR_INVALID_MANIFEST if value doesn't fit in an int
+ */
+int suit_cbor_get_int(const CborValue *it, int *out);
+
+/**
+ * @brief Get cbor value as unsigned
+ *
+ *
+ * @param[in]   it      cbor container iterator
+ * @param[out]  out     address of the returned unsigned
+ *
+ * @return              SUIT_OK on success
+ * @return              SUIT_ERR_INVALID_MANIFEST if value doesn't fit or cannot
+ *                      be converted to unsigned
+ */
+int suit_cbor_get_uint(const CborValue *it, unsigned *out);
+
+/**
+ * @brief Get cbor value as unsigned long
+ *
+ *
+ * @param[in]   it      cbor container iterator
+ * @param[out]  out     address of the returned unsigned long
+ *
+ * @return              SUIT_OK on success
+ * @return              SUIT_ERR_INVALID_MANIFEST if value doesn't fit or cannot
+ *                      be converted to unsigned long
+ */
 int suit_cbor_get_uint32(const CborValue *it, uint32_t *out);
+
+/**
+ * @brief Get cbor value as string
+ *
+ *
+ * @param[in]   it      cbor container iterator
+ * @param[out]  buf     address of the string buffer
+ * @param[out]  len     address of the len of the string
+ *
+ * @return              SUIT_OK on success
+ * @return              SUIT_ERR_INVALID_MANIFEST if value is not a valid string
+ */
 int suit_cbor_get_string(const CborValue *it, const uint8_t **buf, size_t *len);
+
+/**
+ * @brief Parser a cbor subsequence
+ *
+ *
+ * @param[in]   parser      ptr to cbor subparser
+ * @param[out]  bseq        subsequence value
+ * @param[out]  it          cbor iterator
+ *
+ * @return                  0 on success
+ * @return                  -1 if bseq is not a cbor string
+ * @return                  CborError code on other cbor parser errors
+ */
 int suit_cbor_subparse(CborParser *parser, CborValue *bseq, CborValue *it);
 
+/**
+ * @brief Helper function for writing bytes on flash a specified offset
+ *
+ *
+ * @param[in]   arg     ptr to flash writer
+ * @param[in]   offset  offset to write to on flash
+ * @param[in]   buf     bytes to write
+ * @param[in]   len     length of bytes to write
+ * @param[in]   more    whether more data is comming
+ *
+ * @return              0 on success
+ * @return              <0 on error
+ */
 int suit_flashwrite_helper(void *arg, size_t offset, uint8_t *buf, size_t len,
-                    int more);
+                           int more);
```

Comment:
here and below: indent is one off

Reply:
The alignment is now correct with the above line.

Reply:
yes, sorry, in github it looks off.